src/components/MaterialUI/Inputs/DatePicker/index.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import styled, { createGlobalStyle } from "styled-components";
+import { getThemeProp } from "../../../../utils";
+import Icon from "../../../../components/Icon";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import TimePicker from "./TimePicker";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(theme => ({
+	datePickerWrapper: {
+		display: "flex",
+		flexWrap: "wrap",
+		width: "auto",
+		paddingLeft: "5px",
+	},
+}));
+
+const CalendarIcon = styled(Icon).attrs(props => ({
+	id: getThemeProp(["icons", "date"], "calendar")(props),
+}))`
+	font-size: 20px;
+	padding: 3px 3px 3px 3px;
+`;
+
+const DatePickerStyle = createGlobalStyle`
```

Comment:
Please don't use styled components. Use html tags and makeStyles (from material ui) for css

Reply:
Not sure how to convert the createGlobalStyle to a makeStyles...  it took forever to figure out how to override the react-datepicker styles....  I cannot see any samples of override this?